libyul/backends/evm/EVMDialect.cpp

```diff
@@ -170,6 +170,45 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 				_assembly.appendInstruction(evmasm::Instruction::CODECOPY);
 			}
 		));
+		builtins.emplace(createFunction(
+			"setimmutable",
+			2,
+			1,
+			SideEffects{},
+			{true, false},
+			[](
+				FunctionCall const& _call,
+				AbstractAssembly& _assembly,
+				BuiltinContext&,
+				std::function<void()> _visitArguments
+			) {
+				solAssert(_call.arguments.size() == 2, "");
+
+				_visitArguments();
```

Comment:
Hm, we should actually only visit the second argument, right?

Reply:
Yeah, I thought so too. Maybe I need to modify the parameter to allow specifying which one to visit